src/Menu/Filters/LangFilter.php

```diff
@@ -17,10 +17,10 @@ public function __construct(Translator $langGenerator)
     public function transform($item, Builder $builder)
     {
         if (isset($item['header'])) {
-            $item['header'] = ($this->langGenerator->has('adminlte::menu.'.$item['header'])) ? $this->langGenerator->trans('adminlte::menu.'.$item['header']) : $item['header'];
+            $item['header'] = ($this->langGenerator->has('adminlte::menu.'.$item['header'])) ? (version_compare(app()->version(), '6.0.0', '>=') ? $this->langGenerator->get('adminlte::menu.'.$item['header']) : $this->langGenerator->trans('adminlte::menu.'.$item['header'])) : $item['header'];
```

Comment:
Try to swap `app()->version()` with `\Illuminate\Foundation\Application::VERSION` like it's in [page.blade.php L61](https://github.com/jeroennoten/Laravel-AdminLTE/blob/master/resources/views/page.blade.php#L61), that should fix the failed test